covalent_ec2_plugin/ec2.py

```diff
@@ -54,6 +54,12 @@
     }
 )
 
+EC2_KEYPAIR_NAME = "covalent-ec2-keypair"
```

Comment:
Can we maybe add the term `executor` here for verboseness @kessler-frost 🙏🏼  
```suggestion
EC2_KEYPAIR_NAME = "covalent-ec2-executor-keypair"
```